implementations/micrometer-registry-influx/src/main/java/io/micrometer/influx/InfluxMeterRegistry.java

```diff
@@ -211,7 +243,24 @@ protected void publish() {
                 new Field("upper", summary.max())
         );
 
-        return Stream.of(influxLineProtocol(summary.getId(), "histogram", fields));
+        final Stream<Field> totalFields = writePercentiles(summary, fields);
+        return Stream.of(influxLineProtocol(summary.getId(), "histogram", totalFields));
+    }
+
+    private Stream<Field> writePercentiles(HistogramSupport histogramSupport, Stream<Field> fields) {
+        Stream<Field> totalFields;
+        final ValueAtPercentile[] percentileValues = histogramSupport.takeSnapshot().percentileValues();
+        if (percentileValues.length > 0) {
+            List<Field> quantiles = new ArrayList<>(percentileValues.length);
+            for (ValueAtPercentile v : percentileValues) {
+                quantiles.add(new Field("phi" + v.percentile(), v.value(getBaseTimeUnit())));
```

Comment:
what is phi? normally prefix percentiles with just a "p" as like in dropwizard console reporters.

Reply:
To consistent with the prefix of percentiles tag registered in the HistogramGauges.registerWithCommonFormat method

Reply:
This includes value information (the percentile) in the field name, which feels to me like an abuse. Is there some precedent of doing this in InfluxDB?

Reply:
Yes this is how others did thus dropwizard and appmetrics. Though prefix of just p is more typically used.

Reply:
@michaelandrepearce thank you. I see a Dropwizard implementation here as you described it: https://github.com/iZettle/dropwizard-metrics-influxdb/blob/f0d0a9567cedd621630d4a9541459e4e3262aa25/metrics-influxdb/src/main/java/com/izettle/metrics/influxdb/InfluxDbReporter.java

It would be nice if InfluxDB documented a preference for this that different implementations could follow to be aligned.